README.md

```diff
@@ -30,7 +30,8 @@ Once you are ready to create a new release, create a PR to merge main branch to
 Go to https://github.com/strapi/strapi/releases to find the latest version, then update all `@strapi` packages in
 `package.json` to this version and run `npm install`.
 
-# Integration Test
-To run the integration test envrionment, set `NODE_ENV=test` in `.env` file. 
-The login email is `integration-test@devlaunchers.org` and password is `wQ46^BEsVbf9VD&D3ddSPCUP`. 
-Data is stored in `integration-test/data.db`.
\ No newline at end of file
+# Testing
```

Comment:
Is it possible to run tests locally, rigth?

Is there another thing that do we need to setup to run?

I am getting this error, matbe related to `strapi-provider-upload-azure-storage`:


`➜  strapiv4-dev-lauchers-api git:(becevka/playwright-tests) ✗ npm run test

> devlaunchers-strapiv4@0.1.0 test
> playwright test

[WebServer]
> devlaunchers-strapiv4@0.1.0 strapi:test
> strapi develop --no-build

[WebServer] Error: [object Object]
    at makeBlobServiceClient (/Users/tiagohy/my-docs/dev-lauchers/strapiv4-dev-lauchers-api/node_modules/strapi-provider-upload-azure-storage/dist/index.js:48:19)
    at Object.init (/Users/tiagohy/my-docs/dev-lauchers/strapiv4-dev-lauchers-api/node_modules/strapi-provider-upload-azure-storage/dist/index.js:130:31)
    at createProvider (/Users/tiagohy/my-docs/dev-lauchers/strapiv4-dev-lauchers-api/node_modules/@strapi/plugin-upload/server/register.js:60:37)
    at module.exports [as register] (/Users/tiagohy/my-docs/dev-lauchers/strapiv4-dev-lauchers-api/node_modules/@strapi/plugin-upload/server/register.js:16:38)
    at Object.register (/Users/tiagohy/my-docs/dev-lauchers/strapiv4-dev-lauchers-api/node_modules/@strapi/strapi/dist/core/domain/module/index.js:46:46)
    at Object.register (/Users/tiagohy/my-docs/dev-lauchers/strapiv4-dev-lauchers-api/node_modules/@strapi/strapi/dist/core/registries/modules.js:28:19)
    at async Strapi.runLifecyclesFunctions (/Users/tiagohy/my-docs/dev-lauchers/strapiv4-dev-lauchers-api/node_modules/@strapi/strapi/dist/Strapi.js:433:5)
    at async Strapi.register (/Users/tiagohy/my-docs/dev-lauchers/strapiv4-dev-lauchers-api/node_modules/@strapi/strapi/dist/Strapi.js:354:5)
    at async Strapi.load (/Users/tiagohy/my-docs/dev-lauchers/strapiv4-dev-lauchers-api/node_modules/@strapi/strapi/dist/Strapi.js:418:5)
    at async workerProcess (/Users/tiagohy/my-docs/dev-lauchers/strapiv4-dev-lauchers-api/node_modules/@strapi/strapi/dist/commands/actions/develop/action.js:110:26)
Error: Process from config.webServer exited early.

➜  strapiv4-dev-lauchers-api git:(becevka/playwright-tests) ✗`

Reply:
Have you run `npm run init` in that folder before? Do you have `.env` file? In development or test env azure is normally not used.